README.md

```diff
@@ -36,6 +36,15 @@ npm run dev                # Run the development server
 npm run build              # Build the entire site
 ```
 
+This Node.js needs to be version 10.22.0, so you first have to make sure that you're entering these commands with that
+version. To do that, in your bash terminal, enter the following to install that version and use it:
+
+```bash
+nvm install 10.22.0
+```
+
+[nvm](https://heynode.com/tutorial/install-nodejs-locally-nvm) stands for Node Version Manager, and it acts like pipenv for Python, where it allows you to switch between different `Node.js` versions. `Node.js` is a runtime environment that runs websites by executing Javascript code. [npm](https://www.npmjs.com) stands for Node Package Manager, where it keeps track of all of the modules within the web application that `Node.js` recognizes.
```

Comment:
Because the success of the commands above this section depend on the correct version of node, maybe this should be moved above the `npm` commands.

Reply:
```suggestion
[nvm](https://heynode.com/tutorial/install-nodejs-locally-nvm) stands for Node Version Manager, and it acts like pipenv for Python, where it allows you to switch between different `Node.js` versions. `Node.js` is a runtime environment that runs websites by executing Javascript code. [npm](https://www.npmjs.com) stands for Node Package Manager, which keeps track of all of the modules within a web application.
```

Reply:
Also, could "where it keeps track of all of the modules within the web application that `Node.js` recognizes" become "which keeps track of all of the modules within a web application"?

Reply:
I think the main problem I saw with the last sentence was the `where it keeps`, which wasn't changed in your fix. I think `which keeps` would make more grammatical sense. Also, `it acts like pipenv for Python, where it allows` should probably be `it acts like pipenv for Python, which allows`.